tests/test_mssqlcliclient.py

```diff
@@ -202,6 +204,35 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        local_machine_name = socket.gethostname().replace('-','_').replace('.','_')
+        table_name = 'mssqlcli_test_{0}_{1}'.format(local_machine_name, random_str())
```

Comment:
>test_ [](start = 31, length = 5)

we can remove 'test' from the name as it's implied.

Reply:
removed as requested.